cli/cmd/lql_validate.go

```diff
@@ -30,10 +32,10 @@ const (
 var (
 	// queryValidateCmd represents the lql validate command
 	queryValidateCmd = &cobra.Command{
-		Use:   "validate [query_id]",
+		Use:   "validate",
```

Comment:
Removing `query_id` as there's no reason to validate a query from the environment...